book.toml

```diff
@@ -16,4 +16,7 @@ theme = "src/theme"
 
 [output.linkcheck]
 follow-web-links = true
-exclude = [ "www4\\.comp\\.polyu\\.edu\\.hk/~csxluo/" ]
+exclude = [
+	"https://github.com/substrate-developer-hub/recipes/",
```

Comment:
I made almost the same change in the PoW recipe. I wish we didn't have to do it this way, but I don't have any better idea.